pkgs/development/python-modules/pook/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, aiohttp
+, buildPythonPackage
+, fetchFromGitHub
+, furl
+, isPy3k
+, jsonschema
+, mock
+, nose
+, pytestCheckHook
+, requests
+, xmltodict
+}:
+
+buildPythonPackage rec {
+  pname = "pook";
+  version = "1.0.0";
+  disabled = !isPy3k;
+
+  src = fetchFromGitHub {
+    owner = "h2non";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1s2c0g4jis1i6sc2db1v6cr9faib6bcpjlvwnc866z45vsf70lvy";
+  };
+
+  # Separate development and examples from testing requirements
+  patches = [ ./requirements.patch ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    furl
+    jsonschema
+    mock
+    requests
+    xmltodict
+  ];
+
+  checkInputs = [
+    nose
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # Possible issue with Pytest 6
+    "test_base_matcher_exceptions"
```

Comment:
https://github.com/h2non/pook/issues/69
https://github.com/h2non/pook/commit/2071da27701c82ce02b015e01e2aa6fd203e7bb5

Reply:
Thanks, add it as a patch.